packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1102,21 +1102,27 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
       if (!disabled) {
         switch (keyboardKey.getCode(e)) {
           case keyboardKey.Tab:
+            e.stopPropagation();
             this.handleTabSelection(e, highlightedIndex, selectItemAtIndex, toggleMenu);
             break;
           case keyboardKey.ArrowLeft:
+            e.stopPropagation();
```

Comment:
Should we add that to Breaking changes list, in case someone was previously listening to keypress events on dropdown?

Reply:
Added